client/src/sim.rs

```diff
@@ -120,6 +132,14 @@ impl Sim {
         self.jump_pressed = true;
     }
 
+    pub fn set_place_block_pressed_true(&mut self) {
```

Comment:
Good point. I'm not sure whether to address that in this PR for three reasons:
- I already added `set_jump_pressed_true` in a previous PR and probably want to avoid code churn as much as I can in these huge PRs.
- It helps distinguish from `set_jump_held`, which takes a boolean argument.
- This code should be temporary anyway, as we would eventually want to add a layer of abstraction to the input system (although I haven't yet figured out what such an abstraction would look like).

Reply:
No worries.